src/wp-admin/includes/file.php

```diff
@@ -2557,3 +2626,67 @@ function wp_opcache_invalidate( $filepath, $force = false ) {
 
 	return false;
 }
+
+/**
+ * Attempts to clear the opcode cache for a directory of files.
+ *
+ * @since 6.2.0
+ *
+ * @see wp_opcache_invalidate()
+ * @link https://www.php.net/manual/en/function.opcache-invalidate.php
+ * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
```

Comment:
```suggestion
 * @link https://www.php.net/manual/en/function.opcache-invalidate.php
 *
 * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
```

 Per [56070](https://core.trac.wordpress.org/ticket/56070) proposal there should be one line break between `@link` and `@global`

Reply:
Fixed in [c312c67](https://github.com/WordPress/wordpress-develop/pull/3909/commits/c312c675233f0c25a32c772b45bf0591dc0b5129). Thanks Mukesh!